### DISCUSSION TITLE
2021  Databend Sweatshirt is coming

### DISCUSSION THREAD

**Discussion by wubx** [databendlabs/databend#3426] [Announcement]
Our contributor gift has been updated. :tada: 
Before today, we will send you a T-shirt. Starting from Databend 0.7, we plan to send you: Sweatshirt

> Sweatshirt Show

![](https://raw.githubusercontent.com/datafuselabs/datafuse-shop/main/2021-sweatshirt/2021-sweatshirt.png)

![](https://raw.githubusercontent.com/datafuselabs/datafuse-shop/main/2021-sweatshirt/2021-sweatshirt-size.png)

Please provide your subscription to hi@datafuselabs.com like this:

```
Github Nickname: 
Address
name: 
Phone: 
Size: 
```

**Comment by bohutang** [comment]
Thank you list:

@BlackSmith96
@ygf11
@GanZiheng
@koushiro
@bestgopher
@liubaohai 
@Junnplus 
@lianghanzhen
@yihong0618
@kevinw66
@cadl
@jyz0309 
@kevinw66
@tisonkun
@techzealot
@nange
@TennyZhuang
@wangzhen11aaa
@LiuYuHui
@littleeleventhwolf
@Fedomn
@RainnyNightLover
@linyihai
@platoneko
@clark1013
@ZeaLoVe
@chienguo
@july2993
@Milittle
@RinChanNOWWW
@JasonLi-cn
@doki23
@chowc
@jiahui-97
@5kbpers 
@usamoi
@ZuoFuhong

**Comment by bohutang** [comment]
Happy new year!
The first stargazers of the 2022(2022.1.1 easter egg):
@Sunt-ing
@scrogson
@lumost
@forrestneo
@Dowwie
@magicsilence
@navono
@googya
@nimrc
@exherb
